R/iSEE-main.R

```diff
@@ -556,6 +554,29 @@ iSEE <- function(se,
     se
 }
 
+#' Sets the title
+#' 
+#' Prepares the title, either passing what has been provided (and respecting 
+#' any html-based formatting), or setting it to the default, with some info
+#' on the iSEE version in use.
+#' 
+#' @details
+#' Credits to `@RiboRings` for identifying the issue with the previous approach 
+#' based on `ifelse`. Thanks Giulio!
+#'
+#' @param title Character, which can be any value accepted by `dashboardHeader`
+#'
+#' @return The processed title element
+#' 
+#' @rdname INTERNAL_set_title
+.set_title <- function(title) {
+  if( is.null(title) ) {
```

Comment:
The spacing here is inconsistent with the rest of the package - I'd rather do 
`if (is.null(title)) {`

Reply:
oh good catch, odd I did not catch that already. thanks!

Reply:
fixed that now